js/views/homeVw.js

```diff
@@ -470,17 +470,21 @@ module.exports = baseVw.extend({
 
   searchItems: function(searchItemsText){
     if (searchItemsText){
+      var hashedItem = "#" + searchItemsText;
+      
+      window.obEventBus.trigger('saveVisitedTag', hashedItem);
```

Comment:
I agree. The event being fired is "searchItem just happened". It may get used to save the tag, but if some other code wants to know when searchItem occurs, we don't want to add a second event for the same thing.